packages/ui5-info/src/ui5-theme-info.ts

```diff
@@ -0,0 +1,78 @@
+import { defaultVersion } from './constants';
+import type { UI5Theme } from './types';
+import { coerce, gte, lt } from 'semver';
+
+const MIN_UI5_VER_DARK_THEME = '1.72.0';
+const MIN_UI5_VER_HORIZON_THEME = '1.102.0';
+
+export const enum ui5ThemeIds {
+    SAP_BELIZE = 'sap_belize',
+    SAP_FIORI_3 = 'sap_fiori_3',
+    SAP_FIORI_3_DARK = 'sap_fiori_3_dark',
+    SAP_HORIZON = 'sap_horizon',
+    SAP_HORIZON_DARK = 'sap_horizon_dark'
+    // SAP_HORIZON_CONTRAST = 'sap_horizon_hcw',
```

Comment:
Just curious why are these commented out?

Reply:
They were commented out on the internal versions...so I dont know. But Ill remove them now, thanks @cianmSAP .